src/collection/AbstractList.php

```diff
@@ -38,4 +46,63 @@ abstract class AbstractList extends AbstractCollection {
 	public function reverseSort($cmp = null): self {
 		return $this->sort($cmp)->reverse();
 	}
+
+	/**
+	 * Returns the index of the given element or null if the element can't be found
+	 *
+	 * @param mixed $element
+	 *
+	 * @return int the index for the given element
+	 */
+	public function indexOf($element): ?int {
+		$index = $this->parentIndexOf($element);
+
+		return null === $index ? $index : (int) $index;
```

Comment:
Please no yoda style... hard to read it is.

Reply:
:rofl: fixed.